### PR TITLE
Apply Submission and Version models

### LINKED ISSUE #103: Create a django setting that determines the text content of the 'upload a submission' page
Currently, the 'Upload Submission' page does not have customizable text. It should have text that can be customized as a django setting. To allow simpler formatting, this should be markdown rendered.

The current text is:

[^1]: Upload your submission:
Select a zip file, containing the zarr dataset(s) to be evaluated, and upload it. For more detailed instructions check out readthedocs {link}.

Submission requirements:
1. The submission should be a single zip file containing a single Zarr-2 file with the following structure:
   - submission.zarr
     - /<test_volume_name>
        - /<label_name>
2. The names of the test volumes and labels should match the names of the test volumes and labels in the test data.
3. Each label volume should be either A) a 3D binary volume with the same shape and scale as the corresponding test volume, or B) instance IDs per object.
4. The scale for all volumes is 8x8x8 nm/voxel, except as otherwise specified.

## Definition of done

- [ ] A django setting allows setting the text of this page
- [ ] The prod cellmap challenge deployment has the provided text

### PR DESCRIPTION
**Pull request by jnywong**
Projects is renamed to submissions, and submissions is renamed to versions. Each submission can contain a collection of versions, so that users can evaluate multiple versions of a submission so that the best score for a submission is added to the leaderboard.

- My Submissions page is now the central area for users to create, edit and view submissions
- Clicking into an individual submission presents the user with a list of versions submitted and a button to upload a new submission entry
- Add `SubmissionMetadata` as a singleton model and expose this to the Django Admin with the `django-solo` package
    - Closes  #103 
- Use `django-crispy-forms` for submission and upload forms
- Link `Evaluation` to `Version` model
- Link `Version` to `Submission` model
- Modify `page.html` to show header and logo images for all non-Page objects
- Re-use results DataTable to display detailed view of `Evaluation`

**To-do**

- [x] ~~Expose the version name and description fields to the user~~ Decided to use metadata instead
- [x] Link the Version model to the Evaluation model so user's can view the scores
- [x] https://github.com/2i2c-org/unnamed-thingity-thing/issues/103

### CODE REVIEW

File: comptest/web/models.py [hunk 1]
```diff
@@ -24,21 +29,31 @@ class Project(models.Model):
         default=None,
         related_name="projects",
     )
+    gh_repo = models.URLField(max_length=1024, default="https://example.com")
```
Comment (yuvipanda) [inline comptest/web/models.py:32]: This should be something like a “metadata” json field, as different challenges would want different metadata to be collected. And then the list of metadata we wanna collect should be specified as a django setting, perhaps as jsonschema (or just a list for now to meet deadlines)

Comment (jnywong) [reply comptest/web/models.py]: Resolved in https://github.com/2i2c-org/unnamed-thingity-thing/pull/123/commits/4de76606c2d4a3b1a78ed885a453d96d43a4f8ae

**Comment by jnywong** [comment]
Oh yeah, the deployment will need to change `settings.py` the _file/_ prefix for the site logo and site header image because of the fix I made to make these assets available for all non -Page objects.

File: comptest/web/models.py [hunk 2]
```diff
@@ -127,3 +147,22 @@ class ContentFile(models.Model):
 
     def __str__(self):
         return self.title
+
+
+class SubmissionMetadata(SingletonModel):
```
Comment (yuvipanda) [inline comptest/web/models.py:152]: Given that we have a bunch of other settings that should probably be managed via the admin panel (everything with a `SITE` prefix in https://github.com/2i2c-org/unnamed-thingity-thing/blob/b941c59cdc383f7237a56722cb864c5a94842cfe/comptest/comptest/settings.py#L221), can we call this something like `SiteConfiguration` (based on the example in django-solo (https://github.com/lazybird/django-solo)) instead?

Comment (yuvipanda) [reply comptest/web/models.py:152]: This can be done after the fact, merging now to avoid merge conflicts!

**Pull request merged**